pulsar-broker/src/main/java/org/apache/pulsar/broker/TransactionMetadataStoreService.java

```diff
@@ -229,12 +233,24 @@ public void removeTransactionMetadataStore(TransactionCoordinatorID tcId) {
                 completableFutureList.add(actionFuture);
             });
 
+            List<MessageId> messageIdList = new ArrayList<>();
+            for (PulsarApi.MessageIdData messageIdData : messageIdDataList) {
+                messageIdList.add(new MessageIdImpl(
+                        messageIdData.getLedgerId(), messageIdData.getEntryId(), messageIdData.getPartition()));
+            }
+
```

Comment:
Please recycle the Pulsar command after If confirmed no longer use. Please check all.

Reply:
Ok